metaflow/plugins/environment_decorator.py

```diff
@@ -27,5 +27,23 @@ def myStep(self):
     name = 'environment'
     defaults = {'vars': {}}
 
+    def vars_dict(self):
+        env_vars = self.attributes["vars"]
+        if isinstance(vars, str):  # env specified using --with will be a string that we must parse
```

Comment:
```suggestion
        if isinstance(env_vars, str):  # env specified using --with will be a string that we must parse
```

we should really get mypy set up on this :)